src/utils.js

```diff
@@ -57,3 +58,75 @@ export function changeFileName(event, context) {
     tempFileName: event.target.value,
   });
 }
+
+const monthMap = {
+  '01': 'Jan',
+  '02': 'Feb',
+  '03': 'Mar',
+  '04': 'Apr',
+  '05': 'May',
+  '06': 'Jun',
+  '07': 'Jul',
+  '08': 'Aug',
+  '09': 'Sep',
+  10: 'Oct',
+  11: 'Nov',
+  12: 'Dec',
+};
+
+function monthIntToStr(monthInt) {
+  return monthMap[monthInt];
+}
+
+// Takes in a resource file name and prettifies it.
+// =================
+// Each fileName comes in the format of `{date}-{type}-{title}.md`
+// A sample fileName is 2019-08-23-post-CEO-made-a-speech.md
+// {date} is YYYY-MM-DD, e.g. 2019-08-23
+// {type} is either `post` or `download`
+// {title} is a string containing [a-z,A-Z,0-9] and all whitespaces are replaced by hyphens
+export function prettifyResourceFileName(fileName) {
```

Comment:
This only works if existing files are named as such. We get erroneous data because current files do not have type included in their file names, and as a result, title is also missing the first word in the title